server/config/env/development.js

```diff
@@ -12,6 +12,17 @@ module.exports = {
     // Enable mongoose debug mode
     debug: process.env.MONGODB_DEBUG || false
   },
+  orm: {
```

Comment:
I expressed this in a comment below. Should this be a default config setting? Or can this be commented out?